org.adempiere.request/src/main/java/org/compiere/model/RequestModelValidator.java

```diff
@@ -90,6 +93,14 @@ public String login(int AD_Org_ID, int AD_Role_ID, int AD_User_ID) {
     @Override
     public String modelChange(PO entity, int type) throws Exception {
         Boolean isSOTrx = "Y".equals(Env.getContext(entity.getCtx(),"IsSOTrx"));
+        //#2541 Allow to delete draft or in process invoices generated from a request update.
```

Comment:
Hi, it can be replaced by model class wikth a call to Query object instead SQL native

Reply:
Hello Yamel

Originally I thought the same as you, but unfortunately the PO to update only one record, requires loading the entire instance in memory, a result set is executed, and then an update only to remove a reference by value null. is the reason why I do a pure update

Reply:
ok, is very lazy PO for it